.github/workflows/coverage-report.yml

```
@@ -0,0 +1,78 @@
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  build-coverage:
+    runs-on: ubuntu-latest
+    if: ${{ !github.event.pull_request.draft }}
+    steps:
+      - name: Check if PR is created by Dependabot
+        id: dependabot-check
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2193:warning:1:29: The arguments to this comparison can never be equal. Make sure your syntax is correct

(shellcheck)